lib/task_helpers.rb

```diff
@@ -60,6 +60,11 @@ def self.create_index(name=nil)
       end
 
       Elasticsearch::Model.client.indices.create(index: name, body: {mappings: mappings})
+
+      [Comment, CommentThread].each do |model|
+        model.put_search_index_mapping(name)
+      end
```

Comment:
@clintonb: I moved this from app.rb to here.  I first tried to create an index without the body (above), so I didn't need to loop through these classes twice, but the tests around group ids failed.  I'm not actually clear on what is going on, and whether put_search_index_mapping() should have been all that is needed, and whether this is a real issue or not.